Bulldozer.FellowshipOne/Maps/Groups.cs

```diff
@@ -433,61 +645,100 @@ private void MapGroups( IQueryable<Row> tableData, long totalRows = 0 )
                             {
                                 groupType = StripSuffix( groupType, campusId );
                             }
+                            groupType = groupType.Trim();
```

Comment:
Can you move this back to above the campusId, though ironically in the full commit this was added anyway in either spot, so it probably isn't that necessary, your call.